lib/brisket.js

```diff
@@ -43,6 +43,8 @@ var Brisket = {
 
     $: require("./application/jquery"),
 
+    _: require("./application/jquery")._,
```

Comment:
expose Brisket._ so that consumers can still use other underdash type libraries if they want to